### DISCUSSION TITLE
主线图困难图使用周回模式但不使用自律寻敌时如何实现

### DISCUSSION THREAD

**Discussion by guoh064** [LmeSzinc/AzurLaneAutoScript#3340] [General]
目前的主线图功能，在困难图使用周回模式但不使用自律寻敌时，会按照campaign_main中的MAP.spawn_data来搜索敌人。但是困难图周回模式下会直接出现boss，导致搜索敌人失败。
我现在希望利用主线图困难图功能帮助我打捞13-3的杜威，应该如何处理比较好呢？

**Comment by LmeSzinc** [comment]
要捞船去普通图
